src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/Eventing/ParameterCapturingEventSource.cs

```diff
@@ -82,6 +83,9 @@ public void FailedToCapture(Guid RequestId, Exception ex)
         [Event(ParameterCapturingEvents.EventIds.Flush)]
         protected override void Flush()
         {
+            // This method is called on a timer and we shouldn't track any calls inside so we don't flood the user with unimportant captures.
```

Comment:
Let's actually move this to `AbstractMonitorEventSource.FlushTimerTick` so that all of our in-proc event sources pick this up.

Other LGTM